Source/Core/DolphinTool/VerifyCommand.cpp

```diff
@@ -139,7 +139,7 @@ int VerifyCommand(const std::vector<std::string>& args)
   }
 
   // Open the volume
-  const std::unique_ptr<DiscIO::VolumeDisc> volume = DiscIO::CreateDisc(input_file_path);
+  const std::unique_ptr<DiscIO::Volume> volume = DiscIO::CreateVolume(input_file_path);
   if (!volume)
   {
     fmt::print(std::cerr, "Error: Unable to open disc image\n");
```

Comment:
This isn't quite accurate anymore. How about simply:

```suggestion
    fmt::print(std::cerr, "Error: Unable to open file\n");
```